web/packages/teleport/src/Discover/SelectResource/resources.tsx

```diff
@@ -111,20 +111,21 @@ export const APPLICATIONS: ResourceSpec[] = [
 
 export const WINDOWS_DESKTOPS: ResourceSpec[] = [
   {
-    name: 'Active Directory',
+    name: 'Active Directory users',
```

Comment:
For greater consistency with the rest of the tiles (which capitalize all words except in the special case of `macOS`)

```suggestion
    name: 'Active Directory Users',
```